apps/web/.env.example

```
@@ -1,27 +1,80 @@
+# -----------------------------------------------------------------------------
+# Supabase
+# Run `npm run db:start` to get these values from the CLI output.
+# Docker Desktop must be installed and running before starting Supabase.
+# -----------------------------------------------------------------------------
 
-# -- Supabase (local defaults from `supabase start`) -------------------------
 NEXT_PUBLIC_SUPABASE_URL=http://127.0.0.1:54321
-NEXT_PUBLIC_SUPABASE_ANON_KEY=<your-local-anon-key>
-SUPABASE_SERVICE_ROLE_KEY=<your-local-service-role-key>
-SUPABASE_JWT_SECRET=<your-local-jwt-secret>
+NEXT_PUBLIC_SUPABASE_ANON_KEY=<anon key from `npm run db:start` output>
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NEXT_PUBLIC_SUPABASE_ANON_KEY key should go before the NEXT_PUBLIC_SUPABASE_URL key

(UnorderedKey)